pom.xml

```diff
@@ -107,6 +107,13 @@
             <version>3.7.7</version>
             <scope>test</scope>
         </dependency>
+        <!-- To use univocity for csv parsing-->
+        <dependency>
+            <groupId>com.univocity</groupId>
+            <artifactId>univocity-parsers</artifactId>
+            <version>2.9.1</version>
+            <type>jar</type>
```

Comment:
I don't think you need `<type>jar</type>`

Reply:
Resolved in the lasts commits.